.github/workflows/tests-contributors.yml

```diff
@@ -37,30 +37,77 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Generate matrix for test jobs
+        id: generate_matrix
+        run: |
+          FEDVERS=34
+          TARGET_BRANCH=${{ fromJson(steps.pr_api.outputs.data).base.ref }}
+
+          JSON_MASTER_ELN='{"release": "master", "target-branch": "master", "ci-tag": "master"}, {"release": "eln", "target-branch": "master", "ci-tag": "eln", "build-args": "--build-arg=image=quay.io/fedoraci/fedora:eln-x86_64"}'
```

Comment:
I know they are used together but for sake of readability I would split these two.

Reply:
Thanks, done.